projects/observability/src/shared/components/gauge-list/gauge-list.component.ts

```diff
@@ -82,7 +88,7 @@ export class GaugeListComponent<T extends GaugeItem = GaugeItem> implements OnCh
     this.itemOptions = this.items.map(option => ({
       label: option.label,
       color: colorLookupMap.get(option.colorKey ?? option.label),
-      width: `${((option.value / maxValue!) * 100).toFixed(2)}%`,
+      width: `${((option.value / this.total!) * 100).toFixed(2)}%`,
```

Comment:
Now that we have total being supplied from outside, that total needs to be used for percentage calculation in L93 below.
Thoughts?